presto-main/src/main/java/com/facebook/presto/sql/analyzer/AggregationAnalyzer.java

```diff
@@ -110,6 +115,7 @@
 
     private final Metadata metadata;
     private final Analysis analysis;
+    private final Session session;
```

Comment:
move session right after warningCollector to match the order to constructor parameters and assignment